src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -62,6 +66,249 @@ static ShardPlacement * TupleToShardPlacement(TupleDesc tupleDesc,
 /* exports for SQL callable functions */
 PG_FUNCTION_INFO_V1(master_stage_shard_row);
 PG_FUNCTION_INFO_V1(master_stage_shard_placement_row);
+PG_FUNCTION_INFO_V1(citus_table_size);
+PG_FUNCTION_INFO_V1(citus_total_relation_size);
+PG_FUNCTION_INFO_V1(citus_relation_size);
+
+/*
+ * GetShardIntervalsOnNode takes WorkerNode then compare it with each placement of table.
+ * It returns shard intervals of table on that node as a list of shard intervals.
+ * Note that, shard intervals returned as elements of the list are not the copies but the
+ * pointers. DO NOT modify the shard intervals returned by this function.
+ */
+List *
```

Comment:
Let's define all our functions (except user facing ones) static. Also please reorder them such that caller should come before (including user facing ones)